src/App.js

```diff
@@ -1,25 +1,57 @@
-import logo from './logo.svg';
-import './App.css';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import Office from './pages/Office';
+import NavBar from './components/NavBar';
+import Footer from './components/Footer';
+import Reserve from './pages/Reserve';
+import MyReservations from './pages/MyReservations';
+import OfficeDetails from './pages/OfficeDetails';
+import NewOffice from './pages/NewOffice';
+import MyOffices from './pages/MyOffices';
+import Notification from './components/Notification';
+import Welcome from './pages/Welcome';
+import Login from './pages/Login';
+import Signup from './pages/Signup';
+import './index.css';
 
-function App() {
+const App = () => {
```

Comment:
- [ ] Throughout your frontend project, there is lots of old pieces of commented code (too many to enumerate here).  Kindly remove them to improve the cleanliness of the repo. 👍